runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -333,7 +380,10 @@ end
 ---
 --- TODO: Allow for an offset predicate to tailor the injection range
 ---       instead of using the entire nodes range.
+--
 ---@private
+---@return { [string]: { [number]: Region4[] } }
+---           ^ lang      ^ tree_index (sparse)
```

Comment:
might want to use `<pre>` here so that it renders better in :help. Hopefully doxygen can deal with this:

```
---@return
--- <pre>
---        { [string]: { [number]: Region4[] } }
---           ^ lang      ^ tree_index (sparse)
--- </pre>

Reply:
It's `@private`. It doesn't show up in :help, right?